config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml

```diff
@@ -178,7 +178,7 @@ presubmits:
           --timeout=65m
           "--node-args=--image-config-file=${GOPATH}/src/k8s.io/test-infra/jobs/e2e_node/containerd/containerd-release-1.6-presubmit/image-config-presubmit.yaml -node-env=PULL_REFS=$(PULL_REFS)"
 
-  - name: pull-containerd-sandboxed-node-e2e
+  - name: pull-containerd-disable-sandboxed-node-e2e
```

Comment:
A nit on the naming: `pull-containerd-legacy-cri-node-e2e`

Reply:
Done! thanks!